### ISSUE TITLE
Clarify Guidelines 11.3.3.2

### ISSUE THREAD

**Issue by MarjorieBurghart** [TEIC/TEI#1554]
On [Use of the `<gap>`, `<del>`, `<damage>`, `<unclear>`, and `<supplied>` Elements in Combination](http://www.tei-c.org/release/doc/tei-p5-doc/en/html/PH.html#PHCOMB), it is said that: 

> where the text has been rendered completely illegible by deletion or damage and no text is supplied by the editor in place of what is lost: place an empty gap element at the point of deletion or damage. Use the reason attribute to state the cause (damage, deletion, etc.) of the loss of text.

From this, I gather that when some text has been blackened out so thoroughly that one cannot read what it was, the encoding should be: 

`bla bla bla <gap reason="deletion" extent="3 words"/>`

But... For consistency, one might prefer to encode the deletion as a deletion, containing a `<gap>`: 

`bla bla bla <del><gap reason="deletion" extent="3 words"/></del>`

something that would be necessary if one wanted to encode a `<subst>` including an illegible deletion: 

```
bla bla bla <subst>
<del><gap reason="deletion" extent="3 words"/></del>
<add place="above">bla</add>
</subst>
```

Could we clarify the recommendation?

**Comment by hcayless** [comment]
Group at Prague F2F thinks add a sentence to the first bullet to point out that the gap could be wrapped in a del or damage element, and @MarjorieBurghart's example should be added to the examples.

**Comment by scstanley7** [comment]
It looks like @alex-bia already addressed this one (see commits [6ea144a](https://github.com/TEIC/TEI/commit/6ea144a7ff2e684b4269d216fcede6773c06fc6e#diff-aaf09b439e5887f574aea4e8e0ab1ff3) and [2079adf](https://github.com/TEIC/TEI/commit/2079adf88380e44630e12de22a9cc46d8ca7123c#diff-aaf09b439e5887f574aea4e8e0ab1ff3)), so I'm just going to provide a slightly different example and then close this.
